server/test/requests/studentControllers/updateStudentTest.js

```diff
@@ -0,0 +1,91 @@
+require('dotenv').config({ path: 'variables.env' })
+const mongoose = require('mongoose')
+const Student = require('../../../models/Student')
+const chai = require('chai')
+const expect = require('chai').expect
+const sinon = require('sinon')
+const request = require('supertest')
+const serverRequestToScraper = require('request')
+const scraper = require('../../../helpers/scraper')
+const assert = require('assert')
+const app = require('../../../app')
+
+const sandbox = sinon.sandbox.create()
+
+afterEach(() => {
+  sandbox.restore()
+})
+
+describe('PUT /students', () => {
+  beforeEach(done => {
+    const student = new Student({
+      name: 'studentName',
+      username: 'user512',
+      email: 'test@example.com',
+      notes: 'studentNote',
+      completedChallengesCount: 1,
+      completedChallenges: [{}],
+    })
+
+    student.save(() => done())
+  })
+
+  const stubScraper = (error, scraperResponse) => {
```

Comment:
Good point, I tried doing it and uses `import` instead of `require`, so we can a more consistent JS style across frontend and backend but eventually gave up, but yes, we should extract it next time.